### PR TITLE
Standardize on PyTorch for machine learning models

### PR DESCRIPTION
**Pull request by LoyVanBeek**
# Age & Gender recognition
TODO
- [x] Converted code to use .onnx model
- [ ] Launch node from this new package instead of `image_recognition_keras` in `robot_launch_files`? https://github.com/tue-robotics/robot_launch_files/pull/70
- [ ] Better performing model?
    - [ ] Implement some basic dataset/benchmark tailored to our use-case and camera. We're not working with high-quality retouched photos of celebrities as the typical dataset tends to do. MEGA should contain data for this
- [x] Installer file: https://github.com/tue-robotics/tue-env-targets/pull/276

## CI restriction
- [x] Onnxruntime added to rosdistro: https://github.com/ros/rosdistro/pull/32307

### CODE REVIEW

File: image_recognition_pytorch/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Keras image recognition
```
Comment (MatthijsBurgh) [inline image_recognition_pytorch/README.md:1]: Replace all Keras in this document

File: image_recognition_pytorch/test/run_tests.bash [hunk 2]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+nosetests -vv "$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
```
Comment (MatthijsBurgh) [inline image_recognition_pytorch/test/run_tests.bash:2]: Why is this needed? Just for manual testing? As we use the catkin macro.

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]

File: image_recognition_pytorch/README.md (hunk 1 above)
Comment (LoyVanBeek) [reply image_recognition_pytorch/README.md:1]: Crap, I thought I had it all

**Comment by MatthijsBurgh** [comment]
Also we should include `python3-onnxruntime-pip` in rosdistro

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Fix catkin lint
```
image_recognition_pytorch: package.xml: error: unknown package 'python3-onnxruntime-pip'
image_recognition_pytorch: package.xml: error: unknown package 'python3-pytorch-pip'
image_recognition_pytorch: warning: file 'scripts/download_model' is executable but not installed
image_recognition_pytorch: package.xml: notice: meaningless package description 'The image_recognition_pytorch package'
catkin_lint: checked 1 packages and found 4 problems
```

**Pull request closed without merging**